layout.tt

```diff
@@ -91,13 +93,15 @@
                 <li><a href="[%root%]download.html">Download</a></li>
                 <li><a href="[%root%]learn.html">Learn</a></li>
                 <li><a href="[%root%]governance.html">Governance</a></li>
+                <li><a href="[%root%]donate.html">Donate</a></li>
               </ul>
               <ul class="nav pull-right">
                 <li><a href="https://github.com/NixOS/nixpkgs"><i class="fa fa-github"></i></a></li>
                 [% ELSIF menu == 'organization' %]
                 <ul class="nav pull-right">
                   <li><a href="[%root%]learn.html">Learn</a></li>
                   <li><a href="[%root%]governance.html">Governance</a></li>
+                  <li><a href="[%root%]donate.html">Donate</a></li>
```

Comment:
i don't know what site that is, but i added it also next to Learn and Governance